src/core_plugins/kibana/public/dashboard/dashboard_app.js

```diff
@@ -273,14 +274,22 @@ app.directive('dashboardApp', function ($injector) {
         }
 
         confirmModal(
-          `Once you discard your changes, there's no getting them back.`,
+          i18n('kbn.dashboard.changeViewModeConfirmModal.discardChangesDescription',
+            { defaultMessage: 'Once you discard your changes, there\'s no getting them back.' }
```

Comment:
You can use back ticks to prevent apostrophe shielding:
```suggestion
            { defaultMessage: `Once you discard your changes, there's no getting them back.` }
```